app/utilities/schema.py

```diff
@@ -22,7 +22,7 @@
 SCHEMA_DIR = "schemas"
 LANGUAGE_CODES = ("en", "cy")
 
-LANGUAGES_MAP = {"test_language": [["en", "cy"]]}
+LANGUAGES_MAP = [["en", "cy"]]
```

Comment:
This is no longer a language map, has the requirement to have a map by schema name been removed?

During census the map looked something like this:
LANGUAGES_MAP = {
    "test_language": [["en", "cy"], ["en", "ga"]],
    "ccs_household_gb_wls": [["en", "cy"]],
    "census_household_gb_wls": [["en", "cy"]],
    "census_individual_gb_wls": [["en", "cy"]],
    "census_household_gb_nir": [["en"], ["en", "ga"], ["en", "eo"]],
    "census_individual_gb_nir": [["en"], ["en", "ga"], ["en", "eo"]],
    "census_communal_establishment_gb_wls": [["en", "cy"]],
}
Do we not want to do something similar for CRIS/PHM?

Reply:
The requirement hasn't changed but because at the moment we only have the `test_language` line in this. So this means it only sets the correct language for the page in the `test_language` schema. For everything else, every page is "en" even if you launch setting the language to "cy". So this isn't just broken for CRIS and PHM. Considering we only seem to have "en" or "cy" languages at the moment is there the need for a map like this? We could get rid of this and simplify it.

Reply:
Ive had a word with Mebin and realise that we have some tech debt to sort this out so ill just add the broken schemas for now